libs/core/src/lib/token/tokenizer.component.html

```diff
@@ -1,7 +1,20 @@
-<div class="fd-tokenizer" tabindex="0" (focus)="tokenizerHasFocus = true" (blur)="tokenizerHasFocus = false;"
+<div class="fd-tokenizer fd-tokenizer--scrollable" tabindex="0" (focus)="tokenizerHasFocus = true" (blur)="tokenizerHasFocus = false;"
         [ngClass]="{'is-focus': tokenizerHasFocus, 'fd-tokenizer--compact': compact}">
-    <div class="fd-tokenizer__inner">
+    <div class="fd-tokenizer__inner" #tokenizerInner>
         <ng-content select="fd-token"></ng-content>
+        <span *ngIf="(moreTokensLeft.length > 0 || moreTokensRight.length > 0) && compact" class="fd-tokenizer-more" #moreElement>
```

Comment:
I'm not sure if it's good idea to depend behaviour on `compact` flag. Maybe let's have another better docummented flag like `collapseTokens: boolean` or something. It would be great to put some dedicated section only for it

Reply:
Are you saying collapseTokens would be an input or somehow behave separately from the compact flag?  The specs make it pretty clear that phones are cozy and don't have "more" text and desktop is compact and does have "more" so I don't think we should deviate from that